code/_onclick/item_attack.dm

```diff
@@ -93,6 +93,7 @@
 	var/buffout = force*0.25
 	var/smutant = force*0.25
 	var/ghoulmelee = force*0.25 //negative trait, this will cut 25% of the damage done by melee
+	var/power_armor = force*0.25
```

Comment:
No, this combined with bigleagues and buffout would very nearly double the damage of any given melee weapon and make things like super sledges crit you in literally one hit